### PR TITLE
Enhance FAQ entry on green-/low-risk encounters

### LINKED ISSUE #365: Enhance FAQ entry for green/low risk encounters
## Feature description
This is a followup to issue https://github.com/corona-warn-app/cwa-website/issues/307 and PR #319.

The current [FAQ entry](https://www.coronawarn.app/en/faq/#encounter_but_green) on green encounters reads:
>All active Corona-Warn-Apps regularly download the diagnosis keys released on the Corona-Warn-App server and pass them on to the operating system in batches through an interface. The app checks whether any of these received, recorded rolling proximity identifiers match any of the diagnosis keys. If there is a match, this means the following: The user’s smartphone encountered the smartphone of a person who has uploaded a diagnosis key on the day to which the diagnosis key belongs.
>
>In the next step, the app analyzes all the matching rolling proximity identifiers for each diagnosis key, to estimate how long the exposure lasted in total on the day in question and how close the smartphones were to each other on average during the exposure. The distance is calculated from the measured reduction in strength of the Bluetooth signal, which is specified in dB (decibel). All exposures for a diagnosis key that lasted less than 10 minutes in total (regardless of how close the smartphones came during that time) or during which the smartphones were more than 8 meters (73 dB attenuation) apart on average (regardless of how long the exposure lasted) are rated as harmless.
>
>The remaining encounters are rated collectively according to their distance and duration as well as the presumed infectiousness of the other person(s). This rating doesn't always yield an increased risk. This is why the Corona-Warn-App can display encounters, but the risk status stays the same.
>
>For more details about risk assessment, including a sample calculation, see https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md.

I propose the following changes (changes in **bold**):
- "The user’s smartphone encountered the smartphone of a person who has **subsequently** uploaded **their ~~a~~** diagnosis key**s ~~on the day to which the diagnosis key belongs~~**."
The corrected version then reads:  
"The user’s smartphone encountered the smartphone of a person who has subsequently uploaded their diagnosis keys."

Imho the previous version of this statement could be misunderstood to say that the diagnosis key the user matched with was uploaded on the same day the encounter happened which is o/c not the case. This change removes the ambiguity from the statement. Speaking of a specific day in the context of green encounters anyway isn't that useful imho given that green encounters don't show the day on which they happened in CWA.

- "This rating doesn't always yield an increased risk **in which case those encounters - together with the harmless encounters from above - will be shown in the Corona-Warn-App but the risk status will stay the same. ~~This is why the Corona-Warn-App can display encounters, but the risk status stays the same.~~**"
The corrected version then reads: 
"This rating doesn't always yield an increased risk in which case those encounters - together with the harmless encounters from above - will be shown in the Corona-Warn-App but the risk status will stay the same."

This change removes ambiguity on whether "very low risk encounters" (<10min or >73 dB) are shown as "green-/low risk" encounters (which they are) and makes clear that all encounters will be shown at least as green-/low-risk encounters in CWA . 

I've focused on the English version of the FAQ entry for now, but once this version is in a state which makes everyone happy I'd o/c also translate those changes for the [German version](https://www.coronawarn.app/de/faq/#encounter_but_green) 🙂. 

I'd be interested what you think of the proposed changes @svengabr @mtb77 @Ein-Tim 🙂. 
## Problem and motivation
As I [stated](https://github.com/corona-warn-app/cwa-website/pull/319#discussion_r486221652) in the discussion of  #319:
>  The new formulation does indeed fix the repetition, but unfortunately introduces ambiguity whether those "very low risk encounters" (<10min or >73 dB) are shown as "green-/low risk" encounters (which they are). If you're interested I can try to come up with a formulation which unites both aspects later today 🙂

This suggestion should reduce ambiguity in this FAQ entry and give users a clearer picture on how encounters which don't change the overall risk status can happen in CWA.

## Is this something you're interested in working on
Yes. PR: https://github.com/corona-warn-app/cwa-website/pull/366

### PR DESCRIPTION
**Pull request by daimpi**
closes #365 
This PR aims to reduce ambiguity in the FAQ entries for green-/low-risk encounters (cf. #365). 

I've focused on the English version of the FAQ entry for now, but once this version is in a state which makes everyone happy I'd o/c also translate those changes for the German version 🙂.

I'd be interested what you think of the proposed changes @svengabr @mtb77 @Ein-Tim 🙂.

### CODE REVIEW

**Comment by mtb77** [comment]
@daimpi are you finished with this PR? At the moment it is still "work in progress"

Cheers,

Sascha

**Comment by daimpi** [comment]
Hey @mtb77, I'm glad you like it 🙂.
In this case I (or @Ein-Tim) will add the correction for the German version of the FAQ to this PR as well and then I'll mark it "ready for review". Does this sound good to you?

**Comment by mtb77** [comment]
@daimpi @Ein-Tim more then that, sounds great :) Waiting for your update.

Cheers,
Sascha

**Comment by Ein-Tim** [comment]
Added the translation 👍

**Comment by daimpi** [comment]
@mtb77 It's ready for review/merge 🙂.

File: src/data/faq_de.json [hunk 1]
```diff
@@ -58,9 +58,9 @@
...
-                            "Die verbleibenden Begegnungen werden in ihrer Gesamtheit unter Berücksichtigung ihres Abstands und ihrer Dauer sowie der vermutlichen Infektiosität der anderen Person(en) bewertet. Nicht immer ergibt diese Bewertung ein erhöhtes Risiko. Deswegen kann die Corona-Warn-App Ihnen Begegnungen gemeldet haben, ohne dass die Risikobewertung sich ändert.",
+                            "Die verbleibenden Begegnungen werden in ihrer Gesamtheit unter Berücksichtigung ihres Abstands und ihrer Dauer sowie der vermutlichen Infektiosität der anderen Person(en) bewertet. Nicht immer ergibt diese Bewertung ein erhöhtes Risiko, in diesem Fall werden diese Begegnungen - zusammen mit den oben genannten harmlosen Begegnungen - in der Corona-Warn-App angezeigt aber die Risikobewertung ändert sich nicht.",
```
Comment (daimpi) [inline src/data/faq_de.json:63]: one small remark: in the English version we use "harmless" but in the German one it says (above)
> werden als unbedenklich bewertet.

maybe for consistency we should stick with one term throughout the text e.g.
> zusammen mit den oben genannten ~~harmlosen~~ unbedenklichen Begegnungen

Or we change the above to "harmlos"

What do you think? 🙂

Comment (Ein-Tim) [reply src/data/faq_de.json:63]: Let's change above to "harmlos" 👍

**Review by mtb77 — APPROVED** [review]

**Pull request merged**